da/celestia/fees.go

```diff
@@ -0,0 +1,24 @@
+package celestia
+
+const (
+	perByteGasTolerance   = 2
+	pfbGasFixedCost       = 80000
+	defaultGasPerBlobByte = 8
+)
+
+func (c *DataAvailabilityLayerClient) calculateFees(gas uint64) int64 {
+	fees := c.config.Fee
+	if fees == 0 {
+		fees = int64(c.config.GasPrices * float64(gas))
+	}
+
+	return fees
+}
+
+// EstimateGas estimates the gas required to pay for a set of blobs in a PFB.
+func EstimateGas(blobSizes int) uint64 {
+	totalByteCount := 0
```

Comment:
`totalByteCount` seems redundant as we are not summing anything.
Alternative would be to simply change `blobSizes` to `totalBytes` or `blobSize` (we're calculating only for one blob)

Reply:
Also why not make the EstimateGas part of the `DataAvailabilityLayerClient` struct same as `calculateFees` is?